selfdrive/car/chrysler/interface.py

```diff
@@ -113,78 +116,34 @@ def update(self, c, can_strings):
     self.cp.update_strings(can_strings)
     self.cp_cam.update_strings(can_strings)
 
-    self.CS.update(self.cp, self.cp_cam)
-
-    # create message
-    ret = car.CarState.new_message()
+    ret = self.CS.update(self.cp, self.cp_cam)
 
     ret.canValid = self.cp.can_valid and self.cp_cam.can_valid
 
     # speeds
-    ret.vEgo = self.CS.v_ego
-    ret.vEgoRaw = self.CS.v_ego_raw
-    ret.aEgo = self.CS.a_ego
-    ret.yawRate = self.VM.yaw_rate(self.CS.angle_steers * CV.DEG_TO_RAD, self.CS.v_ego)
-    ret.standstill = self.CS.standstill
-    ret.wheelSpeeds.fl = self.CS.v_wheel_fl
-    ret.wheelSpeeds.fr = self.CS.v_wheel_fr
-    ret.wheelSpeeds.rl = self.CS.v_wheel_rl
-    ret.wheelSpeeds.rr = self.CS.v_wheel_rr
-
-    # gear shifter
-    ret.gearShifter = self.CS.gear_shifter
-
-    # gas pedal
-    ret.gas = self.CS.pedal_gas
-    ret.gasPressed = self.CS.pedal_gas > 0
-
-    # brake pedal
-    ret.brake = self.CS.user_brake
-    ret.brakePressed = self.CS.brake_pressed
-    ret.brakeLights = self.CS.brake_lights
-
-    # steering wheel
-    ret.steeringAngle = self.CS.angle_steers
-    ret.steeringRate = self.CS.angle_steers_rate
-
-    ret.steeringTorque = self.CS.steer_torque_driver
-    ret.steeringPressed = self.CS.steer_override
-    ret.steeringRateLimited = self.CC.steer_rate_limited if self.CC is not None else False
+    ret.yawRate = self.VM.yaw_rate(ret.steeringAngle * CV.DEG_TO_RAD, ret.vEgo)
 
-    # cruise state
-    ret.cruiseState.enabled = self.CS.pcm_acc_status  # same as main_on
-    ret.cruiseState.speed = self.CS.v_cruise_pcm * CV.KPH_TO_MS
-    ret.cruiseState.available = self.CS.main_on
-    ret.cruiseState.speedOffset = 0.
-    ret.cruiseState.standstill = False
+    ret.steeringRateLimited = self.CC.steer_rate_limited if self.CC is not None else False
 
     # TODO: button presses
     buttonEvents = []
 
-    if self.CS.left_blinker_on != self.CS.prev_left_blinker_on:
+    if ret.leftBlinker != self.left_blinker_prev:
```

Comment:
Do we need button events for the blinker? We don't use them anywhere? Maybe in a different PR remove them for all cars?

Reply:
yeah, noticed that too. Will remove in another PR.